x-pack/test/scalability/config.ts

```diff
@@ -70,6 +70,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ...(!!AGGS_SHARD_DELAY ? ['--data.search.aggs.shardDelay.enabled=true'] : []),
         ...(!!DISABLE_PLUGINS ? ['--plugins.initialize=false'] : []),
       ],
+      serverArgs: [...baseConfig.kbnTestServer.serverArgs, '--elasticsearch.maxSockets=800'],
```

Comment:
This is here because our current scalability suite doesn't build the distributable from the PR (#151915), so we need to change the config in the FTR suite to test this effect 😇 

We can remove this line once ready to merge